### PR TITLE
Add Label Action

### LINKED ISSUE #101: Add label action (add_input_labels)
Action to let people label an inbound message

### PR DESCRIPTION
**Pull request by ycleptkellan**
resolves #101

### CODE REVIEW

**Comment by ycleptkellan** [comment]
@ericnewcomer ready for review

File: src/config/typeConfigs.ts [hunk 1]
```diff
@@ -98,25 +100,33 @@ export const typeConfigList: Type[] = [
...
         form: SendMsgForm,
         component: SendMsgComp,
         advanced: Mode.EDITING,
         allows
     },
     // { type: 'msg', name: 'Send Message', description: 'Send somebody else a message', form: SendMessageForm, component: SendMessage },
+    {
+        type: Types.add_input_labels,
+        name: 'Add Labels',
+        description: 'Add a label to an incoming message',
```
Comment (ericnewcomer) [inline src/config/typeConfigs.ts]: Label the incoming message

File: src/component/actions/AddLabels/AddLabelsForm.tsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+    constructor(props: AddLabelsFormProps, context: ConfigProviderContext) {
+        super(props);
+
+        bindCallbacks(this, {
+            include: [/^on/]
+        });
+    }
+
+    public onValid(widgets: { [name: string]: any }): void {
```
[the remark this answers is not in the document]
Comment (ycleptkellan) [reply src/component/actions/AddLabels/AddLabelsForm.tsx]: Ahhh, yeah...i guess we never discussed these `onValid` methods that are being called in a static-method kind of way by `NodeEditor`. Should these keep the `on` prefix or `handle` or something else?

**Comment by ycleptkellan** [comment]
alrighty, changes made 👍

File: src/component/actions/AddLabels/AddLabelsForm.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (ericnewcomer) [reply src/component/actions/AddLabels/AddLabelsForm.tsx]: I think we can address these when we continue to evolve NodeEditor later

**Comment by ericnewcomer** [comment]
🔥

File: src/component/actions/AddLabels/AddLabelsForm.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (ycleptkellan) [reply src/component/actions/AddLabels/AddLabelsForm.tsx]: Sounds good 👍

**Pull request merged**